boundedint/int256.go

```diff
@@ -0,0 +1,138 @@
+// Copyright 2020 The Swarm Authors
+// This file is part of the Swarm library.
+//
+// The Swarm library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The Swarm library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the Swarm library. If not, see <http://www.gnu.org/licenses/>.
+
+package boundedint
```

Comment:
I'd like to keep the 256 in the package name. We could just call it `int256` and the types would be `int256.Int256` and `int256.Uint256`. "bounded" is bit generic and I see no reason why types of other sizes would be added.

Reply:
i cannot argue with the points you've made about the name `bounded`.

however, i dislike the redundancy of the `256` number.

i am open to other suggestions

Reply:
package renamed to `int256` and placed inside the `swarm/swap` folder